binding/flutter/lib/porcupine.dart

```diff
@@ -254,27 +176,28 @@ class Porcupine {
     }
   }
 
-  static String _builtInToKeyword(BuiltInKeyword builtIn) {
-    return builtIn
-        .toString()
-        .split('.')
-        .last
-        .replaceAll('_', ' ')
-        .toLowerCase();
-  }
-
   static Future<String> _extractResource(String filePath) async {
```

Comment:
let's change the name of this function to more accurately describe what it's doing. I'm thinking something like "tryExtractFlutterAsset"